tamr_client/datasets/dataframe.py

```diff
@@ -0,0 +1,38 @@
+"""
+Convenient functionality for interacting with pandas DataFrames.
+"""
+
+import json
+
+import pandas as pd
```

Comment:
We need to figure out how to add `pandas` as an optional dependency.

I think we can use [`poetry`'s `extras`](https://python-poetry.org/docs/pyproject/#extras) for this

Reply:
Moved to #351.